server/events/vcs/github_client.go

```diff
@@ -442,7 +442,7 @@ func (g *GithubClient) GetCombinedStatusMinusApply(repo models.Repo, pull *githu
 				if isRequiredCheck(*r.Name, required.RequiredStatusChecks.Contexts) {
 					if *c.Conclusion == "success" {
 						continue
-					} else {
+					} else { //revive:disable-line:superfluous-else
```

Comment:
Instead of adding the // revive, why not remove the else and simply return the `false, nil` on the final line of the function

Reply:
You're right. Sending commit.